src/vmm/src/devices/virtio/vhost_user_block/device.rs

```diff
@@ -230,7 +232,10 @@ impl<T: VhostUserHandleBackend> VhostUserBlockImpl<T> {
             vu_handle,
             vu_acked_protocol_features: acked_protocol_features,
             metrics: VhostUserMetricsPerDevice::alloc(config.drive_id),
-        })
+        };
+        let delta_us = utils::time::get_time_us(utils::time::ClockType::Monotonic) - start_time;
+        vhost_user_blk.metrics.init_time_us.store(delta_us);
```

Comment:
It would also be nice to add time measurement for the activation as it also speaks with the backend over the network.
Just as a note, there is a 3rd interaction that we're adding now: https://github.com/firecracker-microvm/firecracker/pull/4223 , but it originates from the backend, so I can't see a good way of measuring it. Looks like it's up to the backend to provide a metric for that.

Reply:
Just an update: we have changed our mind and we aren't going to support backend messages, so please ignore my previous comment.

Reply:
In my previous comment I referred to the backend-originated messages. A metric for activation time would still be very helpful.

Reply:
ok I can add that

Reply:
added `activate_time_us` to cover this.